devel_subthemer.module

```diff
@@ -281,7 +281,7 @@ function devel_subthemer_process_theme_directory($form_state) {
       'old_dir' => $dirname . '/color',
       'new_dir' => $new_dirname . '/color',
     );
-    devel_subthemer_copy_dir($copy_color_item);
+    devel_subthemer_copy_dir($copy_color_item, $form_state);
```

Comment:
I'm confused. Why are you adding a second parameter to this call to `devel_subthemer_copy_dir()`? The [function](https://github.com/backdrop-contrib/devel_subthemer/blob/150679dd359922cb7a9cc849a6f7e9bf9be078c4/devel_subthemer.module#L385) only takes one parameter, and your issue refers to `devel_subthemer_convert_file()`, not `devel_subthemer_copy_dir()`

Reply:
The problem is with the devel_subthemer_convert_file() function call you'll find within deve_subthemes_copy_dit() function, which needs that second parameter.